resources/js/Pages/Movies/Index.vue

```diff
@@ -1,77 +1,77 @@
 <script setup>
-
-import GuestLayout from '@/Layouts/GuestLayout.vue';
-import { Link, router } from '@inertiajs/vue3';
-import { ref, computed, watch } from 'vue';
-import { debounce } from 'underscore';
-import FeaturedMovie from '@/Movie/FeaturedMovie.vue';
-import MovieList from '@/Movie/MovieList.vue';
-import Paginator from '@/Widgets/Paginator.vue';
+import GuestLayout from "@/Layouts/GuestLayout.vue";
+import { Link, router } from "@inertiajs/vue3";
+import { ref, computed, watch } from "vue";
+import { debounce } from "underscore";
+import FeaturedMovie from "@/Movie/FeaturedMovie.vue";
+import MovieList from "@/Movie/MovieList.vue";
+import Paginator from "@/Widgets/Paginator.vue";
 
 const props = defineProps({
     paginator: {
         required: true,
-        type: Object
+        type: Object,
     },
     searchInput: {
         required: false,
         type: String,
-        default: ''
+        default: "",
```

Comment:
Why do we need to change the quote sign?

Reply:
It's my linter :)